corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -272,23 +272,10 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
 
         iconClick: function (e) {
             e.stopImmediatePropagation();
-            const origin = window.location.origin;
-            const user = FormplayerFrontend.getChannel().request('currentUser');
-            const appId = formplayerUtils.currentUrlToObject().appId;
-            const currentApp = FormplayerFrontend.getChannel().request("appselect:getApp", appId);
-            // Confirms we are getting the app id, not build id
-            const currentAppId = currentApp.attributes["copy_of"] ? currentApp.attributes["copy_of"] : currentApp.attributes["_id"];
-            const domain = user.domain;
             let fieldIndex = $(e.currentTarget).parent().index();
             if (this.isMultiSelect) {
                 fieldIndex -= 1;
             }
-            let caseId;
-            if (this.options.headerRowIndices && !$(e.target).closest('.group-rows').length) {
```

Comment:
That was to support grouped tiles. That seems to be broken now.

Reply:
Correct me if I'm wrong, but putting this back and passing the `caseId` that gets set to `this.clickableIconRequest` instead of  `this.model.get('id')` should fix this, is that right? I don't recall why we were using `this.model.get('id')`.

Reply:
`reloadCase` needs `this.model.get('id')` but the request to form player needs `this.model.get('groupKey');` if the clickable icon is in the grouped section.

Reply:
fixed here: https://github.com/dimagi/commcare-hq/pull/33705/commits/ee8275fd33f04b9fbaa6b9e2b1ab287f63fd6bf4